pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2069,6 +2069,13 @@ public class ServiceConfiguration implements PulsarConfiguration {
     )
     private String loadBalancerLoadSheddingStrategy = "org.apache.pulsar.broker.loadbalance.impl.ThresholdShedder";
 
+    @FieldContext(
+            category = CATEGORY_LOAD_BALANCER,
+            doc = "When 【current usage < average usage - threshold】, "
+                    + "the broker with the highest load will be triggered to unload"
+    )
+    private boolean enableLowerBoundaryShedding = false;
```

Comment:
What if we made the lower boundary a double between 0 and 1 and have it default to 0? Then, the threshold is configurable.

Reply:
What does it mean to set a value between 0 and 1?

Reply:
The value for `LOWER_BOUNDARY_THRESHOLD_MARGIN` is hard coded to 0.5. Instead of enabling this lower bound with a boolean flag, I am proposing that we make the lower boundary configurable.

Reply:
`LoadBalancerBrokerThresholdShedderPercentage` already allows users to control the percentage